scripts/scrape-apple-hig.py

```
@@ -1,472 +1,472 @@
+        help="Show browser window (for debugging)",
+    )
+
+    args = parser.parse_args()
+
+    # Validate start URL is a HIG URL
+    if not is_hig_url(
+        args.start_url,
+        "https://developer.apple.com",
+        "/design/human-interface-guidelines"
+    ):
+        print(f"Error: Start URL must be an Apple HIG URL", file=sys.stderr)
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)